shell/browser/electron_browser_context.h

```diff
@@ -119,8 +119,8 @@ class ElectronBrowserContext : public content::BrowserContext {
 
   void SetUserAgent(const std::string& user_agent);
   std::string GetUserAgent() const;
-  bool CanUseHttpCache() const;
-  int GetMaxCacheSize() const;
+  bool can_use_http_cache() const { return use_cache_; }
+  int get_max_cache_size() const { return max_cache_size_; }
```

Comment:
would we want to remove the `get_` here as well or was there a particular reason this one was left?

Reply:
Thank you for catching these; I agree with all three!

`max_cache_size()` fixed in c248474c0dc995c17955ffa77842e5425a1b104b